scripts/build/parse_blog_tags.py

```diff
@@ -1,9 +1,37 @@
 import json
 import os
 
+release_posts = []
+beta_posts = []
+strip_character = "-"
+
 f = open("src/main/content/blog_tags.json", "r")
 content = json.loads(f.read())
 tags = content['blog_tags']
+
+def format_links_and_remove_duplicates(_posts):
+    final_output = []
+    formatted_posts = []
+    for format_post_name in _posts:
+        date = strip_character.join(format_post_name.split(strip_character)[:3])
+        for idx, x in enumerate(date.split('-')):
+            date = addLeadingZero(date,idx,str(x).zfill(2))
```

Comment:
Request to add some code comments on the intention of this code routine. Some comments showing intent can help with future debugging if this routine performs unexpectedly with future blog posts (unlikely, but doesn't hurt to document the code)

It looks like this is trying to normalize something by adding leading zeros? Is this routine trying to avoid having code that handles different cases in the dates found in previous blog posts that do not conform to majority of the `adoc` files?

Reply:
done